feat(): Bonus 2 with QuestionAnswerAdvisor and SafeGuardAdvisor (#12)
AxaFrance/spring-ai-workshop: README.md

## :gift: Bonus / Best Practices

> 1. [Conversational memory with Advisor](bonus/bonus-1.md) (10min)

Reviewer: ```suggestion
1. [Conversational memory with Advisor](workshop/bonus/bonus-1.md) (10min)
```

Author: done

> 2. [Conversational memory based on vector store (QuestionAnswerAdvisor)](bonus/bonus-2.md) (10min)

Reviewer: ```suggestion
2. [RAG with Advisor (QuestionAnswerAdvisor)](workshop/bonus/bonus-2.md) (10min)
```

Author: done

> 3. [Conversational memory based on vector store (SafeGuardAdvisor)](bonus/bonus-3.md) (10min)

Reviewer: ```suggestion
3. [RAG and safe gard implementation with Advisor (SafeGuardAdvisor)](workshop/bonus/bonus-3.md) (10min)
```

Author: done
